tm/src/main/java/io/seata/tm/api/TransactionalTemplate.java

```diff
@@ -120,17 +136,20 @@ public Object execute(TransactionalExecutor business) throws Throwable {
                 cleanUp();
             }
         } finally {
-            tx.resume(suspendedResourcesHolder);
+            // If the transaction is suspended, resume it.
+            if (suspendedResourcesHolder != null) {
+                suspendedResourcesHolder.resume();
```

Comment:
Why did you make this change?

Reply:
Previously, I thought holder could contain actions, but now it's back.